docs/index.md

```diff
@@ -4,7 +4,141 @@ hide:
   - toc
 ---
 
-<h1 align="center" style="margin-top: 0px;">Dasharo OSF vPub Summer 2022</h1>
-<h2 align="center" style="margin-top: 0px;">More info coming soon!</h2>
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
This is not good, we should serve all fonts because of GPDR.

Reply:
Deleted and fixed affected sections.